### PR TITLE
Update

### PR DESCRIPTION
**Pull request by daison12006013**

### CODE REVIEW

File: config/environments/.env.dev [hunk 1]
```diff
@@ -1,6 +1,5 @@
 APP_ENV="dev"
```
Comment (mosufy) [inline config/environments/.env.dev]: Should standardize environment names with the rest. I.e;
```
APP_ENV="development"
```

File: src/config/jwt.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+   * JWT Secret
+   *--------------------------------------------------------------------------
+   *
+   * Here your jwt secret key. In order to verify a token injected in
+   * "Authorization" header, we should have way to decrypt that token
+   * and validate things.
+   *
+   */
+
+  secret: 'YOUR-JWT-TOKEN-HERE',
```
Comment (mosufy) [inline src/config/jwt.js]: Should be configurable per environment. i.e;

```
process.env.JWT_SECRET=
```

File: src/config/jwt.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+   *--------------------------------------------------------------------------
+   *
+   * Here your issuer validation, if you would like to validate the issuer
+   * you could add the lists of domains below and activate the validation.
+   *
+   */
+
+  iss: {
+    validate: false,
+    data: ['api.domain.com'],
```
Comment (mosufy) [inline src/config/jwt.js]: Should be configurable per environment. i.e;

```
process.env.JWT_ISS_SHOULD_VALIDATE=
process.env.JWT_ISS_DOMAIN=
```

File: src/config/jwt.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+   * the custom claims, you could add the lists of those claims below
+   * and activate the validation.
+   *
+   * Read More: https://auth0.com/docs/tokens/jwt-claims
+   *
+   */
+
+  customClaims: {
+    validate: false,
+    data: ['user'],
```
Comment (mosufy) [inline src/config/jwt.js]: Should be configurable per environment. I.e;

```
process.env.JWT_CUSTOM_CLAIMS_SHOULD_VALIDATE=
process.env.JWT_CUSTOM_CLAIMS_DATA=
```

File: src/middlewares/verifyJwtToken.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  return parsed[1];
+};
+
+const verifyJwtToken = () => {
+  return {
+    before: (handler, next) => {
+      const { headers } = handler.event;
+
+      try {
+        const service = new JwtService(token(headers), config.secret, {
```
Comment (mosufy) [inline src/middlewares/verifyJwtToken.js]: The client does not need to know the technical details to verifying jwt token. Instead, it should be as simple as a single line validation. I.e;

```
import { validateJwtToken } from 'lesgo';

const verifyJwtToken = () => {
  return {
    ...
    return validateJwtToken(token, config.secret, config.options);
  }
}
```

File: config/environments/.env.dev (hunk 1 above)
Comment (daison12006013) [reply config/environments/.env.dev]: Above is changed

File: src/config/jwt.js (hunk 2 above)
Comment (daison12006013) [reply src/config/jwt.js]: Above is changed

File: src/config/jwt.js (hunk 4 above)
Comment (daison12006013) [reply src/config/jwt.js]: Above is changed

File: src/middlewares/verifyJwtToken.js (hunk 5 above)
Comment (daison12006013) [reply src/middlewares/verifyJwtToken.js]: Applied above

File: src/config/jwt.js (hunk 3 above)
Comment (daison12006013) [reply src/config/jwt.js]: Applied above

File: src/middlewares/verifyJwtToken.js (hunk 5 above)
Comment (daison12006013) [reply src/middlewares/verifyJwtToken.js]: Applied above

**Pull request merged**